DeceptionGame/Assets/Scripts/PlayerController.cs

```diff
@@ -1,23 +1,23 @@
-﻿using System.Collections;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerController : MonoBehaviour {
     [Header("References")]
     public ShowOnlyIfInRange showIfInRangeScript;
```

Comment:
Doesn't have to be now, but I think that animation and prop logic should be separated into their own classes, with PlayerController simply having a reference to them (e.g. like PlayerMovementController)